docs/api/api-methods.md

```diff
@@ -0,0 +1,189 @@
+# API Methods
+All API methods calls except for token generation are secured via [OAuth2](oauth.md).
+You need to provide token via `Authorization` header to access requested data like this: ```Authorization: "Bearer eyJ0eXAiOiJKV...s3SKg"```
+
+**Retrieve access token**
+----
+
+* **URL**
+
+  `/api/token`
+
+* **Method:**
+
+  `POST`
+  
+*  **Data Params**
```

Comment:
Please add information that data have to be in `x-www-form-urlencoded` format and there have to be header `Content-Type: application/x-www-form-urlencoded`

Reply:
I do not think it is necessary as it is default "setting" for POST method

Reply:
The header is not required, ok.

But there should be information how to send those parameters in body. On the other hand if you don't want to fix it, ok, this documentation is just temporary